### ISSUE TITLE
[TBC Classic] [Promises, Promises...] quest throws a lua error when accepted

### ISSUE THREAD

**Issue by raethkcj** [Questie/Questie#3387]
## Description
When you accept [Promises, Promises...] Questie throws an error in the chat frame:

```
Quest accepted: Promises, Promises...
Interface\AddOns\Questie\Modules\Map\QuestieMap.lua:546: No UiMapID or fitting parentAreaId for areaId : 3805 - The Map of Zul'Aman
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1028: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1026>
[string "=[C]"]: in function `error'
[string "@Interface\AddOns\Questie\Modules\Map\QuestieMap.lua"]:546: in function `DrawWorldIcon'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:966: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:913>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:794: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:730>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1026: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1024>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1079: in function `PopulateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:404: in function `val'
[string "@Interface\AddOns\Questie\Modules\TaskQueue.lua"]:8: in function <Interface\AddOns\Questie\Modules\TaskQueue.lua:6>

Questie: [ERROR] [QuestieQuest]: v6.8.6 - There was an error populating objectives for Promises, Promises... 11132 1 No error
```
## Wowhead link

https://tbc.wowhead.com/quest=11132/promises-promises

## Questie & WoW version
Questie v6.8.6
WoW Classic TBC 2.5.4.42873

**Comment by Jakobud** [comment]
Also similar errors for the follow up quest `X Marks ... Your Doom!`

**Comment by BreakBB** [comment]
Yeah, I noticed this myself but didn't have the time yet to fix it. Thanks for opening the issue, I will get it fixed next week

**Comment by Cabro** [comment]
Add in Questie\Database\Zones\zoneTables.lua line 197:

    [3805] = {{3433, 82.2, 64.2}},

**Comment by Muehe** [comment]
Fixed in 7f1b5677ae4c31f3636b6b1f7c92a8b3a318bae2

Release will be uploaded shortly™.
